unified_planning/model/problem.py

```diff
@@ -682,6 +686,9 @@ def _kind_factory(self) -> "_KindFactory":
         for goal in chain(*self._timed_goals.values(), self._goals):
             factory.update_problem_kind_expression(goal)
         factory.update_problem_kind_initial_state(self)
+        self.processes
+        if len(list(self.processes)) > 0:
+            factory.kind.set_time("PROCESSES")
```

Comment:
Add also some flags for events!

Reply:
not sure we need to differentiate processes and events. I think a flag (likely not "processes") for both is fine

Reply:
One further comment: if we only have events we need to find a way to understand the semantics. I am not sure this case is well documented in the syntax of PDDL+. For instance, ENHSP considers temporal only a problem where there is at least one process. I don't see any use for events only (as they are understood in PDDL+) as events make sense if triggered by something which is changed as an effect of some passage of time.